cpp/include/cudf/utilities/type_dispatcher.hpp

```diff
@@ -158,6 +159,12 @@ struct type_to_scalar_type_impl<cudf::dictionary32> {
   using ScalarDeviceType = cudf::numeric_scalar_device_view<int32_t>;
 };
 
+template <>
+struct type_to_scalar_type_impl<cudf::list_view> {
+  using ScalarType = cudf::list_scalar;
+  // using ScalarDeviceType = cudf::list_scalar_device_view;
```

Comment:
Is this a TODO?

Reply:
Yes.  It's there just to allow compilation.

Reply:
Please mark it with a TODO comment then.